### DISCUSSION TITLE
Windows security detects trojan

### DISCUSSION THREAD

**Discussion by DeltaBoss** [oMaN-Rod/palworld-save-pal#28] [General]
More specifically Trojan:Win32/Etset!rfn in the newest release psp-windows-0.6.3.zip.

**Comment by Lyriati** [comment]
Same.

**Comment by oMaN-Rod** [comment]
Compiling has been a pain, I've been testing pyinstaller, Nuitka, and cx_Freeze with varying results. Seems the best option will be ditching single file compilation, I'm also switching to cx_Freeze for the next release since it raises the least number of false positives.
